### ISSUE TITLE
A11y_Azure Data Studio_Connect to a SQL Server_High Contrast: Windows is not adapting when the high contrast desert theme is applied.

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18555]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG2.1; #Win32; #AINotInScope; #HighContrast;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1. Turn on High Contrast desert theme on Settings.
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Hover over the 'Servers' header and select the 'New Connection' icon.
5. Provide server and windows authentication details, then click 'Connect'.
6. Validate in left menu that server has been successfully connected.
7. Verify that whether the high contrast desert theme is adapting or not.

### Actual Result:
Windows is not adapting when the high contrast Desert theme is applied.

### Expected Result:
Windows should be adapted when the high contrast Desert theme is applied.
### Attachment:
![A11y_Azure Data Studio_Connect to a SQL server_High Contrast](https://user-images.githubusercontent.com/99962579/155731188-fc00da45-e3c5-4eb0-955c-48aba09af59c.gif)

**Comment by alanrenmsft** [comment]
@Gayathri1501 we will keep the issue open and let you guys close it. for this specific one, please close it ASAP, it will show up on our s360 dashboard.

**Comment by Gayathri1501** [comment]
Verified this issue in the environment. Hence, closing this bug
https://user-images.githubusercontent.com/99962579/178259966-0aca2fc6-2001-4623-ac72-d0f594a82792.gif

**Comment by SachinChavhan317** [comment]
#Closed

**Comment by Samhitha66** [comment]
#Closed;
